Slingshot.Api/Abstract/Repository.cs

```diff
@@ -130,6 +131,15 @@ public virtual string UserName
             return null;
         }
 
+#pragma warning disable 1998
+        public async virtual Task<bool> IsPrivate()
+        {
+            // Default we assume repo is public repo
+            _isPrivate = false;
```

Comment:
I'd still argue that this assignment is useless since no code will ever look at it.